.gitignore

```diff
@@ -81,3 +81,6 @@ lib/
 src/icons/
 .out
 .size-snapshot.json
+
+# Typescript declaration types generated 
+.tempTypes
```

Comment:
Based on your research and work, how do you think we should be generating these types? It seems like it is standard to commit the generated `.d.ts` files alongside the associated source file. Was `.tempTypes` to help you create/delete with ease?

Reply:
this folder is the outDir for the types in the config. As I said the generator was not providing anything useful and it just cluttered all the codebase I had to manually delete all generated `d.ts` files from the `src` folder. I have looked at other project declarations files and all of them looks like they use custom declaration written by hand.

Besides the generator would put the `d.ts` files inside `src` folder and we dont want that we want that we are not publishing that

Reply:
not need this temp folder anymore. deleting...